api/api.go

```diff
@@ -420,12 +431,32 @@ type CloudBackupScheduleInfo struct {
 	// MaxBackups are the maximum number of backups retained
 	// in cloud.Older backups are deleted
 	MaxBackups uint
+	// GroupID indicates the group of volumes for this cloudbackup schedule
+	GroupID string
+	// Labels indicates a volume group for this cloudsnap schedule
+	Labels map[string]string
 }
 
 type CloudBackupSchedCreateRequest struct {
 	CloudBackupScheduleInfo
 }
 
+type CloudBackupGroupSchedCreateRequest struct {
```

Comment:
The schedules will be deleted with the same API right? Maybe use CloudBackupSchedCreateRequest (and path) for creating both too?

Reply:
Yes. Same API is used to delete the schedule as both are recognized by UUID. thought of re-suaing the same API, but input parameters could get confusing. Hence the separate API.